cogs5e/initTracker.py

```diff
@@ -784,13 +785,8 @@ def mod_or_set(opt_name, old_value):
         args = argparse(args)
 
         targets = []
-        first_target = await combat.select_combatant(target_name)
-        if first_target is None:
-            await ctx.send("Combatant not found.")
```

Comment:
So, in my testing this line would never run. `NoSelectionElements` was always returned before it could reach the if statement.